cmd/libs/go2idl/generator/generator.go

```diff
@@ -110,6 +110,10 @@ type Generator interface {
 	// Generators.)
 	Init(*Context, io.Writer) error
 
+	// Finalize should write finish up functions, and any other content that's not
```

Comment:
Sorry, what do you mean by "finish up functions"? Probably giving an example will help?